### PR TITLE
Add documentation for new bandwidth pricing

### PR DESCRIPTION
**Pull request by gargakshit**
### Summary of changes

Add documentation for the new pricing model for bandwidth.

### Preview

image

### CODE REVIEW

File: about/pricing.html.markerb [hunk 1]
```diff
@@ -203,19 +203,30 @@ We use Lets Encrypt to issue certificates, and donate half of our SSL fees to th
...
+- data transfer to some [extensions](https://fly.io/docs/about/pricing/#extensions) like Supabase or Upstash Redis (excludes Tigris Object Storage)
 
-Any traffic exiting a Fly.io Machine is considered outbound data transfer, including:
+You're not billed for the following types of traffic:
 
-* traffic destined for the internet
-* traffic destined for other Machines or apps in your network
-* traffic destined for [extensions](#extensions) like Supabase or Upstash Redis
+- all inbound data transfer
+- data transfer between apps or Machines in the same region
```
Comment (andie787) [inline about/pricing.html.markerb:217]: @gargakshit do we need another sentence here like "(for orgs created after July 17 2024)"?

Comment (gargakshit) [reply about/pricing.html.markerb:217]: I guess that would make sense.

```suggestion
- data transfer between apps or Machines in the same region (for organizations created after July 17 2024)
```

File: about/pricing.html.markerb [hunk 2]
```diff
@@ -203,19 +203,30 @@ We use Lets Encrypt to issue certificates, and donate half of our SSL fees to th
...
-* traffic destined for [extensions](#extensions) like Supabase or Upstash Redis
+- all inbound data transfer
+- data transfer between apps or Machines in the same region (for organizations created after July 17 2024)
+- data transfer from apps without an assigned IP address (for orgs created before July 17 2024)
+- data transfer to [Tigris Object Storage](https://fly.io/docs/reference/tigris)
 
-One exception: traffic destined for [Tigris Object Storage](/docs/reference/tigris) is free. Learn more about [Extensions billing](#extensions).
+Fly.io pricing is per region group for outbound data transfer. You'll see a more detailed breakdown of cost per region and per traffic type on your monthly invoice.
+
+> **Important**: Organizations created after July 17 2024 are billed at a different rate for private network data transfer between regions, per the following table. Organizations created before July 17 2024 are billed for all data transfer (excluding that listed as free above) at the "Egress to public internet" rate.
```
Comment (andie787) [inline about/pricing.html.markerb:223]:
```suggestion
<div class="important icon">
**Important:** Organizations created after July 17 2024 are billed at a different rate for private network data transfer between regions, per the following table. Organizations created before July 17 2024 are billed for all data transfer (excluding that listed as free above) at the "Egress to public internet" rate.
</div>
```

File: about/pricing.html.markerb [hunk 3]
```diff
@@ -203,19 +203,30 @@ We use Lets Encrypt to issue certificates, and donate half of our SSL fees to th
...
 
-Any traffic exiting a Fly.io Machine is considered outbound data transfer, including:
+You're not billed for the following types of traffic:
 
-* traffic destined for the internet
-* traffic destined for other Machines or apps in your network
-* traffic destined for [extensions](#extensions) like Supabase or Upstash Redis
+- all inbound data transfer
+- data transfer between apps or Machines in the same region (for organizations created after July 17 2024)
+- data transfer from apps without an assigned IP address (for orgs created before July 17 2024)
```
Comment (andie787) [inline about/pricing.html.markerb:218]: just to be consistent with the one above
```suggestion
- data transfer from apps without an assigned IP address (for organizations created before July 17 2024)
```

**Review by andie787 — COMMENTED** [review]
left a few comments. Also, are we leaving in the bit about Tigris data transfer being free?

**Comment by gargakshit** [comment]
> Also, are we leaving in the bit about Tigris data transfer being free?

I'll fix it ASAP once this is rolled out. But I guess we can add that in later.

**Review by andie787 — APPROVED** [review]

**Comment by andie787** [comment]
@gargakshit You are good to merge when you're ready or to your schedule :)

**Pull request merged**